server/src/services/media.service.spec.ts

```diff
@@ -797,6 +873,21 @@ describe(MediaService.name, () => {
       );
     });
 
+    it('should transcode when max bitrate is not a number', async () => {
+      mediaMock.probe.mockResolvedValue(probeStub.videoStream40Mbps);
+      systemMock.get.mockResolvedValue({ ffmpeg: { transcode: TranscodePolicy.BITRATE, maxBitrate: 'foo' } });
```

Comment:
This should probably be a config validation error.

Reply:
Agreed, but then https://github.com/immich-app/immich/blob/main/server/src/services/media.service.ts#L465-L467 wouldn't be necessary correct?

Reply:
Yeah, that would be redundant at that point